pom.xml

```diff
@@ -27,6 +27,8 @@
 		<org.springframework.cloud.version>4.0.0</org.springframework.cloud.version>
 		<com.playtika.reactivefeign.version>3.2.6</com.playtika.reactivefeign.version>
 		<io.netty.version>4.1.89.Final</io.netty.version>
+		<org.springframework.kafka.version>3.0.2</org.springframework.kafka.version>
+		<io.projectreactor.kafka.version>1.3.16</io.projectreactor.kafka.version>
 	</properties>
 
 	<dependencies>
```

Comment:
Dependency "org.springframework.kafka" is needed for TopicBuilder

Reply:
I got this error from idea: "Unable to load io.netty.resolver.dns.macos.MacOSDnsServerAddressStreamProvider, fallback to system defaults. This may result in incorrect DNS resolutions on MacOS. Check whether you have a dependency on 'io.netty:netty-resolver-dns-native-macos"

Reply:
its a local dependency, so its should be removed

Reply:
Ok